frontend/javascripts/admin/organization/upgrade_plan_modal.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useRef } from "react";
+import { Divider, InputNumber, Modal } from "antd";
+import moment from "moment";
+import {
+  DatabaseOutlined,
+  FieldTimeOutlined,
+  RocketOutlined,
+  UserAddOutlined,
+} from "@ant-design/icons";
+import { APIOrganization } from "types/api_flow_types";
+import { formatDateInLocalTimeZone } from "components/formatted_date";
+import {
+  sendExtendPricingPlanEmail,
+  sendUpgradePricingPlanEmail,
+  sendUpgradePricingPlanStorageEmail,
+  sendUpgradePricingPlanUserEmail,
+} from "admin/admin_rest_api";
+import { powerPlanFeatures, teamPlanFeatures } from "./pricing_plan_utils";
+import { PricingPlanEnum } from "./organization_edit_view";
+
+function extendPricingPlan(organization: APIOrganization) {
+  const extendedDate = moment(organization.paidUntil).add(1, "year");
+
+  Modal.confirm({
+    title: "Extend Current Plan",
+    okText: "Request Extension",
+    onOk: sendExtendPricingPlanEmail,
+    icon: <FieldTimeOutlined style={{ color: "var(--ant-primary-color)" }} />,
+    width: 1000,
+    content: (
+      <div
+        style={{
+          background:
```

Comment:
yes it was meant to overlap and be at the edge. Moving it under the main text make it hard to read.

Reply:
Ok, I see. Positioning is fine by me. I'd reduce the transparency a bit maybe, since the "We" in the footer is a bit hard to read.

> also, it's quite edge on my screen. maybe it can be smoothed somehow?

With "quite edge" I meant "quite edgy" or pixelated. Not sure how the quality can be improved, since it's probably a svg?